hazelcast/src/main/java/com/hazelcast/internal/yaml/YamlLoader.java

```diff
@@ -51,6 +51,8 @@ private YamlLoader() {
     public static YamlNode load(InputStream inputStream, String rootName) {
         try {
             Object document = getLoad().loadFromInputStream(inputStream);
+            inputStream.close();
```

Comment:
Is it possible to close this one and the ones below on the same method where it is created? To avoid further problems such as when adding new methods.

Reply:
Good point, thanks :+1: It can and should be closed in the config builders.

Reply:
@mmedenjak Moved closing the YAML InputStream to `YamlConfigBuilder` from `YamlLoader`. The `YamlLoader#load(Reader)` methods are not used in the production code, just removed the `reader.close()` from those methods.